anax-in-container/service/anax.service

```diff
@@ -80,6 +80,28 @@ defaultEnv() {
 	  #checkrc $? "change ServiceStorage"
 	#fi
 
+	# setup fss domain socket path and ess auth path
+	if [[ -z "${DOCKER_NAME}" ]]; then
+		DOCKER_NAME="horizon1"
+	fi
+	if [[ "${HOST_OS}" == "mac" ]]; then
+		base_host_dir="/private/var/tmp/horizon/${DOCKER_NAME}"
+		mkdir -p /private/var/tmp
+		ln -s /var/tmp/horizon /private/var/tmp/horizon
+	else
+		base_host_dir="/var/tmp/horizon/${DOCKER_NAME}"
+	fi
+
+	if [[ -d /var/tmp/horizon/${DOCKER_NAME}/ess-auth ]]; then
```

Comment:
Since you are going to set the config field to `$base_host_dir/ess-auth`, shouldn't that be what you check for in the if statement.

Reply:
This is in within the container, both mac and others have `/var/tmp/horizon/${DOCKER_NAME}/ess-auth`

Reply:
Oh well, maybe checking `$base_host_dir/ess-auth` and `$base_host_dir/fss-domain-socket` is more clear.